### DISCUSSION TITLE
Posts not showing up

### DISCUSSION THREAD

**Discussion by lmilbaum** [mmistakes/minimal-mistakes#4298] [Q&A]
I've tried so many options for hours with no avail so far.
On the other hand, I get a "Recent Posts" header.
Would appreciate your assistance.

https://github.com/devopsloft/devopsloft.github.io

**Comment by lmilbaum** [comment]
NM. Solved it.

**Reply by xerxes-k** [reply]
how?

**Reply by lmilbaum** [reply]
I don't remember what solved it. Feel free to pick into my code devopsloft/devopsloft.github.io
